mobile_config/src/boosted_hex_info.rs

```diff
@@ -295,7 +316,7 @@ mod tests {
         };
 
         let msg = BoostedHexInfo::try_from(proto)?;
-        assert_eq!(631252734740306943, msg.location);
+        assert_eq!(hextree::Cell::from_raw(631252734740306943)?, msg.location);
```

Comment:
why not import hextree:Cell in the above crate and remove the need to fully qualify each usage of Cell

Reply:
I think this file was one of the early ones in the change, before I felt like `Cell` was used enough to live on it's own. I've updated the occurrences in this file. :+1: